src/tasks/copy-template-files.ts

```diff
@@ -16,61 +16,40 @@ const EXTERNAL_EXTENSION_TMP_FOLDER = "tmp-external-extension";
 const copy = promisify(ncp);
 let copyOrLink = copy;
 
-const expandExtensions = (options: Options): Extension[] => {
-  const expandedExtensions = options.extensions
-    .map(extension => extensionDict[extension])
-    .map(extDescriptor => [extDescriptor.extends, extDescriptor.value].filter(isDefined))
-    .flat()
-    // this reduce just removes duplications
-    .reduce((exts, ext) => (exts.includes(ext) ? exts : [...exts, ext]), [] as Extension[]);
-
-  return expandedExtensions;
-};
```

Comment:
Checkout this => https://github.com/scaffold-eth/create-eth/pull/45#discussion_r1639970569